vale/dictionary.jsonnet

```
@@ -205,6 +205,7 @@ local newWord(word, affixes, po) = {
     newWord('subnet', 'S', 'noun'),
     newWord('subquery', 'S', 'noun'),
     newWord('subtask', 'S', 'noun'),
+    newWord('SUT', '', 'noun') { acronym: true },
```

I think it's useful to explain the abbreviation as in    

```
newWord('Relational Database Service', '', 'noun') { Amazon: true, description: 'Amazon Relational Database Service', product: true },
```

```suggestion
    newWord('SUT', '', 'noun') { acronym: true, description: 'System Under Test', },
```

---

I agree it's good to have a description here! Thanks for adding it. 🙇